src/main/resources/application.properties

```diff
@@ -18,8 +18,10 @@ spring.main.sources=org.entur.kishar
 kishar.incoming.port=8888
 
 # URL to be used in Camel to fetch data
-kishar.anshar.polling.url.et=http4://anshar/anshar/rest/et
+kishar.anshar.polling.url.et=http4://localhost:8776/anshar/rest/et
 kishar.anshar.polling.url.vm=http4://anshar/anshar/rest/vm
 kishar.anshar.polling.url.sx=http4://anshar/anshar/rest/sx
 
 kishar.anshar.polling.period=15s
+
+kishar.datasource.whitelist=RUT
```

Comment:
Noen operatører har ikke synkronisert sine SIRI-tjenester, så det bør være én whitelist per datatype for sikkerhets skyld - selv om de stort sett vil være like...